examples_src/vector-esri.js

```diff
@@ -0,0 +1,113 @@
+goog.require('ol.Attribution');
+goog.require('ol.Map');
+goog.require('ol.View');
+goog.require('ol.format.EsriJSON');
+goog.require('ol.layer.Tile');
+goog.require('ol.layer.Vector');
+goog.require('ol.loadingstrategy');
+goog.require('ol.source.Vector');
+goog.require('ol.source.XYZ');
+goog.require('ol.style.Stroke');
+goog.require('ol.style.Style');
+goog.require('ol.tilegrid.XYZ');
+
+
+var esrijsonFormat = new ol.format.EsriJSON();
+
+var vectorSource = new ol.source.Vector({
+  attributions: [
+    new ol.Attribution({
+      html: 'Bron: &copy; 2012, Centraal Bureau voor de Statistiek' +
+          ' / Kadaster, Zwolle, 2012'
+    })
+  ],
+  loader: function(extent, resolution, projection) {
+    var url = 'http://services.arcgis.com/nSZVuSZjHpEZZbRo/arcgis/rest/' +
+        'services/CBS_Gemeenten_2013_RD/FeatureServer/0/query?f=json&' +
+        'returnGeometry=true&spatialRel=esriSpatialRelIntersects&geometry=' +
+        encodeURIComponent('{"xmin":' + extent[0] + ',"ymin":' +
+            extent[1] + ',"xmax":' + extent[2] + ',"ymax":' + extent[3] +
+            ',"spatialReference":{"wkid":102100}}') +
+        '&geometryType=esriGeometryEnvelope&inSR=102100&outFields=*' +
+        '&outSR=102100&callback=loadFeatures';
+    // use jsonp: false to prevent jQuery from adding the "callback"
+    // parameter to the URL
+    $.ajax({url: url, dataType: 'jsonp', jsonp: false});
+  },
+  strategy: ol.loadingstrategy.tile(new ol.tilegrid.XYZ({
+    tileSize: 512
+  }))
+});
+
+// the global function whose name is specified in the URL of JSONP ArcGIS
+// REST Feature Service requests
+var loadFeatures = function(response) {
```

Comment:
You probably don't need to use a global function in this case, and let jQuery create the callback function for you. The case of GeoServer (`vector-wfs` example) is different because the callback is specified in a specific way.